browser/ui/webui/webcompat_reporter/webcompat_reporter_ui.cc

```diff
@@ -114,6 +115,106 @@ void WebcompatReporterDOMHandler::RegisterMessages() {
       "webcompat_reporter.submitReport",
       base::BindRepeating(&WebcompatReporterDOMHandler::HandleSubmitReport,
                           base::Unretained(this)));
+  web_ui()->RegisterMessageCallback(
+      "webcompat_reporter.captureScreenshot",
```

Comment:
just wondering what processes can use `chrome.send` to send these messages? let's say for instance brave://newtab is compromised by one of the external resources it loads through brave news; can the malicious script use `chrome.send` to get a screenshot capture?

Reply:
message handlers are registered for the current web UI instance only. since other internal pages have their own individual web UI instances, sending a message from other pages will be a no-op.